### PR TITLE
prepare to be the default StepZen template on Vercel

### PR DESCRIPTION
**Pull request by vlukashov**
- update README
- make use of StepZen GraphQL API in the app

How to test (after the PR is merged):
 - click the "Deploy to Vercel" button
 - follow the process on Vercel and StepZen dashboard
 - see that the app gets deployed
 - clone the repo locally
 - follow the steps in README
 - see that the app starts locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,72 @@
...
+   ```bash
+   npm i -g stepzen
+   stepzen login
+   ```
+
+   and create a `.env.local` file to pass your StepZen credentials to the NextJS app:
+
+   ```bash
+   echo "STEPZEN_ACCOUNT=$(stepzen whoami --account)" >> .env.local
+   echo "STEPZEN_API_KEY=$(stepzen whoami --apikey)" >> .env.local
```
Comment (royderks) [inline README.md:20]: there seems to be no mention of `STEPZEN_ENDPOINT` here?

File: README.md [hunk 2]
```diff
@@ -1,36 +1,72 @@
...
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [StepZen Documentation](https://stepzen.com/docs) - learn about StepZen features and API.
 - [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+You can check out [the StepZen `examples` GitHub repository](https://github.com/stepzen-dev/examples) for more StepZen examples.
+
+## Deploy to Vercel
+
+If you created this app by clonning the starter repo with the "Deploy to Vercel" button, your Vercel project already has the `STEPZEN_ACCOUNT` and `STEPZEN_API_KEY` environment variables correctly defined. The StepZen Vercel integtaion does that for you, and you do not need to make any extra actions.
```
Comment (royderks) [inline README.md]: there seems to be no mention of `STEPZEN_ENDPOINT` here?

File: README.md (hunk 1 above)
Comment (vlukashov) [reply README.md:20]: updated README

File: README.md [hunk 3]
```diff
@@ -1,36 +1,100 @@
...
+---
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+<img width="1306" alt="demo" src="https://user-images.githubusercontent.com/22416150/203358874-7ccbe260-0f70-4433-9340-974dd6e17616.png">
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+Open [`http://localhost:3000`](http://localhost:3000) with your browser to see the app, and [`dashboard.stepzen.com`](https://dashboard.stepzen.com/) to see your GraphQL API endpoint.
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+You can edit the GraphQL API my modifying `stepzen/index.graphql` by hand, or use the [`stepzen import`](https://stepzen.com/docs/cli/cli-commands#stepzen-import) CLI command to import additional data sources to your API.
```
Comment (SamuelGHill) [inline README.md]: my --> by

File: README.md [hunk 4]
```diff
@@ -1,36 +1,100 @@
...
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+The page auto-updates as you edit JS or CSS files, and the GraphQL API on `stepzen.net` auto-updates as you edit GraphQL schema files in `stepzen/`.
+
+The GraphiQL component on the home page is only an example to show that your app has a GraphQL API. Feel free to delete this component at any time. You can always use your [StepZen dashboard](https://dashboard.stepzen.com/) to explore your GraphQL endpoints.
```
Comment (SamuelGHill) [inline README.md]: Non-blocker:

https://dashboard.stepzen.com/ --> https://dashboard.stepzen.com/explorer

File: README.md [hunk 5]
```diff
@@ -1,36 +1,100 @@
...
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
+- [StepZen Documentation](https://stepzen.com/docs) - learn about StepZen features and API.
 - [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+You can check out [the StepZen `examples` GitHub repository](https://github.com/stepzen-dev/examples) for more StepZen examples.
+
+## Deploy to Vercel
+
+If you created this app by clonning the starter repo with the "Deploy to Vercel" button, your Vercel project already has the `STEPZEN_ACCOUNT` and `STEPZEN_API_KEY` environment variables correctly defined. The StepZen Vercel integtaion does that for you, and you do not need to make any extra actions.
```
Comment (SamuelGHill) [inline README.md]: "environment variables correctly defined" --> "environment variables correctly defined in your vercel project" with a link?

Comment (SamuelGHill) [reply README.md]: https://vercel.com/docs/concepts/projects/environment-variables

**Review by SamuelGHill — APPROVED** [review]

**Pull request merged**